domains/rikubot.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "devops-riku",
+        "email": "egermino.riku@gmail.com"
+    },
+    "record": {
+        "CNAME": "rikudev.cloud"
```

Comment:
seems to just redirect to Hostinger page, can you add your website here?

Reply:
so you dont have a website hosted atm?

Reply:
you need _something_ to be able to register

Reply:
> so you dont have a website hosted atm?

yeah, exactly

Reply:
kindly check my site again, rikudev.cloud, I hosted my test page, thank you

Reply:
I don't have write access but there's something now

Reply:
No worries, Ill wait for the approval thankss